code/__HELPERS/unsorted.dm

```diff
@@ -1767,3 +1767,19 @@ Game Mode config tags:
 			continue
 		turfs += T
 	return pick(turfs)
+
+/proc/get_closest_atom(type, list, source)
```

Comment:
I ended not using this, not sure if i leave it or nah 🤔

Reply:
delet